src/inverseRobot.js

```diff
@@ -24,17 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  const arrKeys = Object.values(robot);
-  const arrValues = Object.keys(robot);
+  const robotKeys = Object.values(robot);
+  const robotValues = Object.keys(robot);
   const newRobot = {};
-  const double = [];
+  const errCheck = [];
 
-  for (let i = 0; i < arrKeys.length; i++) {
-    if (double.includes(arrKeys[i])) {
+  for (let i = 0; i < robotKeys.length; i++) {
+    if (errCheck.includes(robotKeys[i])) {
```

Comment:
in checklist 2, there is an advice how can we make check if some keys repeate. there shoud be newRobot.hasOwnProperty(robotKeys[i]) i think

Reply:
thanks)) I think, I didn't quite understand this point. But I'll try)